src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkWindows.cs

```diff
@@ -163,6 +162,37 @@ private static IEnumerable<string> GetOpenJdkPaths ()
 			}
 		}
 
+		/// <summary>
+		/// Locate OpenJDK installations by well known path.
+		/// </summary>
+		/// <returns>List of valid OpenJDK paths in version descending order.</returns>
+		private static IEnumerable<string> GetKnownOpenJdkPaths ()
+		{
+			string JdkFolderNamePattern = "microsoft_dist_openjdk_";
+
+			var paths = new List<Tuple<string, Version>> ();
+			var rootPaths = new List<string>
+			{
+				Path.Combine (Environment.ExpandEnvironmentVariables ("%ProgramW6432%"), "Android", "Jdk"),
```

Comment:
On my machine, this is `%ProgramW6432%\Android\jdk`, not `...\Jdk`.

Reply:
I don't think case matters on Windows, tested in Workbooks:

![image](https://user-images.githubusercontent.com/2809202/59117502-e758f580-891b-11e9-8d02-071e584d5633.png)

Reply:
Apparently it _can_ matter if someone is crazy enough to enable such on Windows. Updated anyway.